src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/CollectionFloatingLineplotPlugin.java

```diff
@@ -0,0 +1,127 @@
+package org.veupathdb.service.eda.ds.plugin.standalonemap;
+
+import org.gusdb.fgputil.ListBuilder;
+import org.gusdb.fgputil.validation.ValidationException;
+import org.veupathdb.service.eda.common.client.spec.StreamSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.ConstraintSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.DataElementSet;
+import org.veupathdb.service.eda.common.plugin.util.PluginUtil;
+import org.veupathdb.service.eda.ds.Resources;
+import org.veupathdb.service.eda.ds.plugin.AbstractEmptyComputePlugin;
+import org.veupathdb.service.eda.ds.plugin.AbstractPlugin;
+import org.veupathdb.service.eda.ds.plugin.standalonemap.markers.OverlaySpecification;
+import org.veupathdb.service.eda.generated.model.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.veupathdb.service.eda.common.plugin.util.PluginUtil.singleQuote;
+import static org.veupathdb.service.eda.common.plugin.util.RServeClient.streamResult;
+import static org.veupathdb.service.eda.common.plugin.util.RServeClient.useRConnectionWithRemoteFiles;
+import static org.veupathdb.service.eda.ds.metadata.AppsMetadata.VECTORBASE_PROJECT;
+
+public class CollectionFloatingLineplotPlugin extends AbstractEmptyComputePlugin<FloatingLineplotPostRequest, FloatingLineplotSpec> {
+  private OverlaySpecification _overlaySpecification = null;
+
+  @Override
+  public String getDisplayName() {
+    return "Line plot";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Visualize aggregate values of one variable across the sequential values of an ordered Variable Group";
+  }
+
+  @Override
+  public List<String> getProjects() {
+    return List.of(VECTORBASE_PROJECT);
+  }
+
+  @Override
+  public ConstraintSpec getConstraintSpec() {
+    return new ConstraintSpec()
+      .dependencyOrder(List.of("yAxisVariable"), List.of("xAxisVariable", "overlayVariable"))
+      .pattern()
+        .element("xAxisVariable")
+          .shapes(APIVariableDataShape.ORDINAL, APIVariableDataShape.CONTINUOUS)
+          .description("Variable must be ordinal, a number, or a date and be the same or a child entity as the variable the map markers are painted with, if any.")
```

Comment:
the map markers must be painted for this line plot, no? Otherwise what would the y axis be?

Reply:
Sure.. I thought the map had a bunch of defaults though.. maybe I'm misunderstanding you?

Reply:
> be the same or a child entity as the variable the map markers are painted with, if any.

This sounds to me like there might not be a map marker variable? For this map type i thought there was always a marker var?